ReverseProxy.php

```diff
@@ -45,13 +45,15 @@ public function __invoke(Request $req, Response $res) {
 		unset($headers["accept-encoding"]);
 		$connection = $headers["connection"];
 		unset($headers["connection"]);
-		foreach ($connection as $value) {
-			foreach (explode(",", strtolower($value)) as $type) {
-				$type = trim($type);
-				if ($type == "upgrade") {
-					$headers["connection"][0] = "upgrade";
-				} else {
-					unset($headers[$type]);
+		if (isset($connection)) {
```

Comment:
This should be `isset($headers["connection"])` and then `$connection = $headers["connection"]` inside that.